README.md

```diff
@@ -101,6 +108,13 @@ Option to remove the component level folders. This would turn `/lib/jquery/js/jq
 
 **Note:** If your components have files with the same name then only one of them will be included in the results.
 
+####options.typeTop
+
+Type: `boolean`
+Default: `false`
+
+Option to put the type folder on top of the hierarchy. This would turn `/lib/jquery/js/jquery.js` into `/lib/js/jquery/jquery.js`.
+
```

Comment:
Where is the functionality for this option added? I'm looking at this diff and I don't see anything to address it.